aiida/backends/sqlalchemy/models/node.py

```diff
@@ -43,18 +44,16 @@ class DbNode(Base):
     __tablename__ = 'db_dbnode'
 
     id = Column(Integer, primary_key=True)  # pylint: disable=invalid-name
-    uuid = Column(UUID(as_uuid=True), default=get_new_uuid, unique=True)
-    node_type = Column(String(255), index=True)
-    process_type = Column(String(255), index=True)
-    label = Column(
-        String(255), index=True, nullable=True, default=''
-    )  # Does it make sense to be nullable and have a default?
-    description = Column(Text(), nullable=True, default='')
-    ctime = Column(DateTime(timezone=True), default=timezone.now)
-    mtime = Column(DateTime(timezone=True), default=timezone.now, onupdate=timezone.now)
+    uuid = Column(UUID(as_uuid=True), default=get_new_uuid, nullable=False)
+    node_type = Column(String(255), default='', nullable=False)
+    process_type = Column(String(255))
```

Comment:
default?

Reply:
the default here, since `process_type` is nullable, will automatically be `None`